fe/fe-core/src/test/java/com/starrocks/sql/parser/ParserTest.java

```diff
@@ -302,6 +319,22 @@ private static Stream<Arguments> multipleStatements() {
         sqls.add(Pair.create("select 1 xxx select 2 xxx", false));
         return sqls.stream().map(e -> Arguments.of(e.first, e.second));
     }
+
+    private static Stream<Arguments> setQuantifierInAggFunc() {
```

Comment:
Why not add cases in SQL-test which can get real results?

Reply:
We already have these integration tests in StarRocksSqllogicTest project. So I decide only add some FE UT.